lib/DDG/Spice/Maven.pm

```diff
@@ -4,15 +4,22 @@ package DDG::Spice::Maven;
 use strict;
 use DDG::Spice;
 
-triggers startend => "maven", "mvn";
-
+triggers any => "maven", "mvn","repository","repo","java library","package","library java","lib java","java lib","module java","java module";
```

Comment:
@jeyadossmanimaran these are too broad. The triggers we should be added are the ones outlined in the issue:

`maven repo`, `maven repository`, `maven package`, `java package`.

We can keep `java lib` and `java module` as well

Please also update the test file with tests that ensure these work as expected 👍